coalib/output/ConsoleInteraction.py

```diff
@@ -505,7 +505,7 @@ def acquire_settings(log_printer, settings_names_dict):
 
     result = {}
     for setting_name, arr in sorted(settings_names_dict.items(),
-                                    key=lambda x: join_names(x[1][1:])):
+                                    key=lambda x: (join_names(x[1][1:]), x[0])):
```

Comment:
Yes that is what I wanted, `x[1][1:]` has the bear name, so I want it to be the first key, and then alphabetically sort the settings of that bear, so `x[0]`